specs/simpleserialize.md

```diff
@@ -0,0 +1,230 @@
+# SimpleSerialize (SSZ) Spec
+
+***Work In Progress***
+
+This is the work in progress document to describe `simpleserialize`, the
+current selected serialization method for Ethereum 2.0 using the Beacon Chain.
+
+This document specifies the general information for serializing and
+deserializing objects and data types.
+
+## ToC
+
+* [About](#about)
+* [Terminology](#terminology)
+* [Constants](#constants)
+* [Overview](#overview)
+ + [Serialize/Encode](#serializeencode)
+   - [uint: 8/16/24/32/64/256](#uint-816243264256)
+   - [Address](#address)
+   - [Hash32](#hash32)
+   - [Bytes](#bytes)
+   - [List](#list)
+ + [Deserialize/Decode](#deserializedecode)
+   - [uint: 8/16/24/32/64/256](#uint-816243264256-1)
+   - [Address](#address-1)
+   - [Hash32](#hash32-1)
+   - [Bytes](#bytes-1)
+   - [List](#list-1)
+* [Implementations](#implementations)
+
+## About
+
+`SimpleSerialize` was first proposed by Vitalik Buterin as the serializaiton
```

Comment:
typo: `serializaiton` -> `serialization`

Reply:
Good catch! Completely missed the misspelling :smile: Thanks!